docs/pages/reference/aa-sdk/core/functions/webauthnGasEstimator.mdx

```diff
@@ -0,0 +1,55 @@
+---
+# This file is autogenerated
+title: webauthnGasEstimator
+description: Overview of the webauthnGasEstimator method
+slug: wallets/reference/aa-sdk/core/functions/webauthnGasEstimator
+---
+
+A middleware function to estimate the gas usage of a user operation when using a Modular Account V2 WebAuthn account. Has an optional custom gas estimator.
+This function is only compatible with accounts using EntryPoint v0.7.0, and the account must have an implementation address defined in `getImplementationAddress()`.
+
+## Import
+
+```ts
+import { webauthnGasEstimator } from "@aa-sdk/core";
+```
+
+## Usage
+
+```ts twoslash
```

Comment:
hmm usually when this section doesn't actually get formatted, there's an error in the example code. That will cause twoslash to throw errors (if we ever get that enabled)